test/FunctionalTests/Microsoft.OData.Core.Tests/ValidationHelper.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.OData.Tests
+{
+    public class ValidationHelper
+    {
+        /// <summary>
+        /// This method takes 2 generic types and see if one is the deep copy  of other
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="obj"></param>
+        /// <param name="copy"></param>

```

Comment:
If you add the comments, finish the comments

Reply:
Resolved